tests/dualtor/test_toggle_mux.py

```diff
@@ -77,8 +87,4 @@ def test_toggle_mux_from_cli(duthosts, tbinfo, active_side, get_mux_status, get_
     mux_active_dut.shell('config muxcable mode active all')
 
     check_result = wait_until(60, 5, 2, check_mux_status, duthosts, active_side)
-
-    if cur_interval_v4 is not None:
-        update_linkmgrd_probe_interval(duthosts, tbinfo, cur_interval_v4)
-
```

Comment:
I thought we were supposed to change the interval back to 1000ms instead of 100ms at the end of all tests? 

https://github.com/sonic-net/sonic-mgmt/blob/f6a9c4fd7d3cea2db2ac18fb5420d5c455d43012/tests/common/dualtor/dual_tor_utils.py#L1360-L1365

Reply:
Yes, `reset_link_prober_interval_v4` will do that